app/views/ex_links/edit.html.haml

```diff
@@ -0,0 +1,7 @@
+%h1 Editing ex_link
+
+= render 'form'
+
+= link_to 'Show', @ex_link
```

Comment:
The `link_to` here is nice, but it is not considered nice by our code-climate since it could be a security issue.
Prefer using full paths: `ex_link_path(@ex_link)`